tailormap-components/projects/core/src/lib/feature-form/form-creator/form-creator-helpers.ts

```diff
@@ -1,18 +1,31 @@
-import { Feature } from '../../shared/generated';
+import { Field, Feature } from '../../shared/generated';
 import { FeatureAttribute, FormConfiguration, IndexedFeatureAttributes } from '../form/form-models';
+import { AttributeListFeature } from '../../shared/attribute-service/attribute-models';
+import { FormHelpers } from '../form/form-helpers';
 
 export class FormCreatorHelpers {
 
-  public static convertFeatureToIndexed(feat: Feature, formConfig: FormConfiguration): IndexedFeatureAttributes {
+  public static convertFeatureToIndexed(feat: Feature | AttributeListFeature, formConfig: FormConfiguration): IndexedFeatureAttributes {
     const m = new Map<string, FeatureAttribute>();
     for (const field of formConfig.fields) {
+      let value = null;
```

Comment:
Suggestion: move this to helper function as well since its duplicate code here and in form-tree-helpers.ts